src/main/java/net/sf/jabref/gui/FileExtensions.java

```diff
@@ -0,0 +1,61 @@
+package net.sf.jabref.gui;
+
+public enum FileExtensions {
+
+    //importannt: No dot before the extension!
+    BIBTEX_DB(
+            "BibTeX Database", "bib"),
+    BIBTEXML(
+            "BibTexMl, bibx"),
+    AUX(
+            "Aux file", "aux"),
+    SILVER_PLATTERN(
+            "SilverPlatter", "dat"),
+    PUBMED(
+            "Pubmed file", "fcgi"),
+    RIS(
+            "Ris file ", "ris"),
+    ENDNOTE(
+            "Endnote/Refer file", "ref"),
+
+    JSTYLE(
+            "Style file", "jstyle"),
+    LAYOUT(
+            "Custom Layout File", "layout"),
+    XML(
+            "XML file", "xml"),
+    TERMS(
+            "Protected TERMS file", "terms"),
+    TXT(
+            "Plain text file", "txt"),
+    CLASS(
+            "Class file", "class"),
```

Comment:
I was not sure about this, so I wanted to wait for feedback. But I agree that it probably makes sense.